src/models/AnonymousIP.ts

```diff
@@ -2,12 +2,38 @@ import mmdb = require('maxmind');
 
 /** Class representing the model of an "AnonymousIP" response **/
 export default class AnonymousIP {
+  /**
+   * The IP address that the data in the model is for. If you performed a "me"
+   * lookup against the web service, this will be the externally routable IP
+   * address for the system the code is running on. If the system is behind a
+   * NAT, this may differ from the IP address locally assigned to it.
+   */
   public ipAddress?: string;
+  /**
+   * Returns true if the IP address belongs to any sort of anonymous network.
```

Comment:
```suggestion
   * True if the IP address belongs to any sort of anonymous network.
```

"Returns" on these is a bit weird given that it is an property rather than a method.

Same with below.